docs/development/releasing.rst

```diff
@@ -432,59 +434,51 @@ release. The rest of the procedure is the same as any other release as
+`astropy-tools repository`_, in the ``pr_consistency`` directory.  These scripts
+essentially check that the above two conditions are met. Detailed documentation
+for these scripts is given in their repository, but here we summarize the basic
+workflow.  Run the scripts in order (they are numbered 1.<something>.py,
+2.<something>.py, etc.), entering your github login credentials as needed (if
+you are going to run them multiple times, using a ``~/.netrc`` file is
+recommended). The script to actually scheck consistency should be run as:
 
-    $ ./backport.sh
+    $ python 4.check_consistency.py > consistency.html
 
-This will checkout the appropriate bug fix branch ('v1.2.x' in this example),
-do a ``git pull upstream v1.2.x`` to make sure it's up to date, and then start
-doing cherry-picks into the bug fix branch.
+Which will generate a simple web page that shows all of the areas where either
+a pull request was merged into master but is *not* in the relevant release that
+it has been milestoned for, as well as any changelog irregularities (i.e., PRs
+that are in the wrong section for what the github milestone indicates).  You'll
+want to correct those irregularities *first* before starting the backport
+process (re-running the scripts in order as needed).
 
-.. note::
+The end of the ``consistency.html`` page will then show a series of
+``git cherry-pick`` commands to update the maintainence branch with the PRs that
+are needed to make the milestones and branches consistent.  Make sure you're in
+the correct maintainence branch with e.g.,
 
-    As discussed earlier, cherry-pick may result in merge conflicts.  If this
-    occurs, the ``backport.sh`` script will exit and the conflict should be
-    resolved manually, followed by running ``git commit``.  To resume the
-    ``backport.sh`` script after the merge conflict, it is currently necessary
-    to edit the script to either remove or comment out the ``git cherry-pick``
-    commands that already ran successfully.
+    $ git checkout v1.3.x
 
-    The author of the script hopes to improve it in the future to add
-    ``git rebase`` like functionality, such that running
-    ``backport.sh --continue`` or ``backport.sh --skip`` will be possible in
-    such cases.
+if you are doing bugfixes for the 1.3.x series. Go through the commands one at a
+time, following the cherry-picking procedure described above. If for some reason
+you determine the github milestone was in error and the backporting is
+impossible, re-label the issue on github and move on.
 
 .. warning::
 
-    It has also been noted that the ``suggest_backports.py`` script is not
-    perfect, and can either miss issues that need to be backported, and in some
-    cases can report false positives.
+    Automated scripts are never perfect, and can either miss issues that need to
+    be backported, or in some cases can report false positives.
 
     It's always a good idea before finalizing a bug fix release to look on
```

Comment:
I've run into another issue today: if we do a feature release, all the still open ``Affects-dev`` issues/PRs need to get their labels changes to ``Affects-release``. Otherwise this could trick astrobot, and us, too.